.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "root": true,
+  "extends": ["./node_modules/sanctuary-style/eslint-es3.json"],
```

Comment:
Why ES3 rather than ES6?

```suggestion
  "extends": ["./node_modules/sanctuary-style/eslint-es6.json"],
```

Reply:
To support older browsers, no?

Reply:
Sadly I can't, because I'm using es6 modules. The idea is to use es6 modules without any other es6 features, so that if users concatenate the code into a bundle using rollup or similar, they can use that bundle in old run-times.

Reply:
I suspected you would have a good reason. :)